app-text/sword/sword-1.8.1.recipe

```diff
@@ -54,33 +63,43 @@ REQUIRES_devel="
 
 BUILD_REQUIRES="
 	haiku${secondaryArchSuffix}_devel
+	icu${secondaryArchSuffix}_devel
 	devel:libcrypto$secondaryArchSuffix
 	devel:libcurl$secondaryArchSuffix
+	devel:libclucene_core$secondaryArchSuffix
+	devel:libnghttp2$secondaryArchSuffix
 	devel:libssl$secondaryArchSuffix
-	devel:libz$secondaryArchSuffix
 	"
 BUILD_PREREQUIRES="
 	cmd:awk
-	cmd:gcc$secondaryArchSuffix
+	cmd:cppunit_config$secondaryArchSuffix
 	cmd:icu_config$secondaryArchSuffix
+	cmd:gcc$secondaryArchSuffix
 	cmd:ld$secondaryArchSuffix
 	cmd:make
 	cmd:pkg_config$secondaryArchSuffix
 	"
 
 BUILD()
 {
-	runConfigure ./configure --without-conf --without-clucene
+	runConfigure ./configure
 	make $jobArgs
 }
 
 INSTALL()
 {
-	make $jobArgs install
+	make install
+
+	rm "$libDir"/libsword.la
```

Comment:
Suggestion:
```diff
-	rm "$libDir"/libsword.la
+	rm -f "$libDir"/libsword.la
```
(This way, if the build tools are taught, some day, to stop creating the libtool files then this recipe will not break suddenly, with all the other recipes which still don't use `-f`.)